tools/wizards/mnecpp/example/main.cpp

```diff
@@ -67,8 +67,8 @@
  */
```

Comment:
Add `#include <utils/generics/applicationlogger.h>`

Reply:
https://github.com/mne-tools/mne-cpp/blob/d979ea1c1f66500fea7818454e8bd8f52fa2a4b5/tools/wizards/mnecpp/example/main.cpp#L38

I think it is already included?

Reply:
sry did not see this. my bad.